### PR TITLE
Added step for importing streams flow file

### PR DESCRIPTION
**Pull request by emlaver**

### CODE REVIEW

File: streams-designer/README.md [hunk 1]
```diff
@@ -45,15 +45,31 @@ These are the IBM Bluemix services required to run this tutorial:
 Cloud Object Storage instance was not created beforehand, you will have the option of adding one.
 ![Create New Project in Watson Studio](create-new-project-watson-studio.png) 
 
-## Create a Streams flow
+## Option 1: Import the Streams flow file
+1. Create a new Streams flow under the `Add to project` drop-down list.
+![Select Streams flow](streams-flow-watson-studio.png)
+1. On the **New Streams Flow** screen, Select **From file**.  Drag and drop [Cloudant_to_Db2_streams_flow.stp](Cloudant_to_Db2_streams_flow.stp)
```
Comment (ricellis) [inline streams-designer/README.md:51]: Doesn't this file need to be uploaded too?

File: streams-designer/README.md [hunk 2]
```diff
@@ -45,15 +45,31 @@ These are the IBM Bluemix services required to run this tutorial:
...
 1. Create a new Streams flow under the `Add to project` drop-down list.
 ![Select Streams flow](streams-flow-watson-studio.png)
 1. On the **New Streams Flow** screen, add a name and optional description for the project. 
 The Streaming Analytics service drop-down box should already be filled in.  
 Select the **Manually** box, then press **Create**.
 ![Create Streams flow](create-streams-flow-watson-studio.png)
 
-## Create operators in the Streams Designer editor
+## Create the Code operator
+**Note:** Skip this section if you imported the streams file.
```
Comment (ricellis) [inline streams-designer/README.md]: Instead of the note to skip this, I think it would be better to make this more steps under option 2.

**Review by ricellis — CHANGES_REQUESTED** [review]

File: streams-designer/README.md (hunk 1 above)
Comment (emlaver) [reply streams-designer/README.md:51]: Yes that would be helpful 😄 .  Added in e19db09.

File: streams-designer/README.md (hunk 2 above)
Comment (emlaver) [reply streams-designer/README.md]: Moved steps in 28577cb.

**Review by ricellis — APPROVED** [review]

**Pull request merged**